### PR TITLE
Neilj/mau limits

### PR DESCRIPTION
**Pull request by neilisfragile**
Creates an endpoint under admin/ to expose current MAU usage and server imposed limits to admin users

### CODE REVIEW

File: changelog.d/3510.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+Expose monthly active user data via _matrix/client/api/v1/admin/server_metrics
```
Comment (turt2live) [inline changelog.d/3510.feature]: r0 not v1, surely? (also, should the admin endpoints be specced?)

Comment (neilisfragile) [reply changelog.d/3510.feature]: ah yes, good spot

**Comment by neilisfragile** [comment]
Following out of band conversation - will solve this by adding the same data via the prometheus endpoint. Will go back and redo.

**Comment by krombel** [comment]
I understand the metrics aspect of adding this data. But what is the intention behind `limit_mau` and `default_max_mau`?

And as "MAU" is nowhere described: It stand for "Monthly Active Users" if I got it right.
So how you can limit that?

**Comment by neilisfragile** [comment]
@krombel yup mau means monthly active user - it ties in with a broader feature that will let homeserver administrators set limits on usage of their homeserver via an optional setting

**Comment by richvdh** [comment]
> Will go back and redo.

I assume this means neil is continuing to work on it and it is no longer ready for review

**Comment by neilisfragile** [comment]
Closing, superseded by #3630

**Pull request closed without merging**